scripts/get

```diff
@@ -130,7 +130,11 @@ downloadFile() {
 # installs it.
 installFile() {
   HELM_TMP="/tmp/$PROJECT_NAME"
-  local sum=$(openssl sha -sha256 ${HELM_TMP_FILE} | awk '{print $2}')
+  local sha_cmd='sha1'
+  if ! openssl list-message-digest-commands | grep --silent '^sha1$' ; then
```

Comment:
So this command isn't working for me on Debian:
```shell
root@12641688a9ee:/# openssl version
OpenSSL 1.1.0f  25 May 2017
root@12641688a9ee:/# openssl list-message-digest-commands
Invalid command 'list-message-digest-commands'; type "help" for a list.
```

In 1.1, I think the command has changed to: `openssl list -digest-commands`

Reply:
Thank you for testing on Debian. I did test the original hack in a Debian one but neglected to retest there. I will check some more distros/versions and figure something out.